SSH into to a Kubernetes cluster (#897)
OctopusDeploy/blog: blog/2022-11/ssh-into-k8s-cluster/index.md

---
title: SSH into a Kubernetes cluster
description: Learn how to set up a SSH bastion host in your Kubernetes cluster.
author: matthew.casperson@octopus.com
visibility: public
published: 2022-11-09-1400
metaImage: blogimage-testingkubernetes-2022.png
bannerImage: blogimage-testingkubernetes-2022.png
bannerImageAlt: Kubernetes logo on an open laptop screen
tags:
 - DevOps
 - Kubernetes
 - Docker
---

Jump boxes or bastion hosts are a common networking strategy to expose a single secure entry point to the public internet, to access a private network. This single point of entry lets security teams closely monitor and control network access to the private network. Often the bastion host exposes a well known remote access service, like RDP or SSH, which teams can assume have been widely vetted and are trustworthy.

> In this post, you learn how to host an OpenSSH server in a Kubernetes cluster to perform administrative tasks.

Reviewer: ```suggestion
In this post, I explain how to host an OpenSSH server in a Kubernetes cluster to perform administrative tasks.
```
